main.go

```diff
@@ -26,7 +27,11 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	log.Println("running...")
+	port := os.Getenv("PORT")
```

Comment:
Checks the PORT environment variable and uses that if set, otherwise awoo (42300)
Also organized it to output the port it bound to correctly before switching to the log output, giving the person running it some indication that the server started.